### PR TITLE
Adds dispensary roofs

### PR DESCRIPTION
**Pull request by curstwist**
#### Summary
```SUMMARY: Content "Adds roof to 3 dispensary shops, adds 2 new nested maps"```

#### Purpose of change
Adds dispensary roofs and access.  Adds beaded curtain to 2 blueprints. 

Once I was looking at the `dispensary.json`, I noticed one was a duplicate of dispensary_2, it looked like an earlier version so I removed it.  Also, fixed counters to use t_floor instead of t_wall. 

Dependent on Adds antique store roof #28859 for roof_3x3_wine mapgen
#### Additional context
dispensary 
dispensary
dispensary roof

dispensary_1
dispensary_1a
dispensary_roof_1

dispensary_2
dispensary_2
dispensary_roof_2b

nested maps:
roof_2x2_utilities_c
roof_2x2_utilities_b

stairs added to `dispensary.json`
stairs added roof
stairs added

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
I can't find a stairs or ladder up to the roof on the `dispensary` location. Is this intentional?

File: data/json/mapgen/dispensary.json [hunk 1]
```diff
@@ -448,5 +449,66 @@
...
+        "4": "t_gutter_north",
+        "5": "t_gutter_drop",
+        "#": "t_grate",
+        ">": "t_ladder_down"
+      },
+      "furniture": { "&": "f_roof_turbine_vent", ":": "f_vent_pipe" },
+      "place_items": [ { "item": "roof_trash", "x": [ 4, 15 ], "y": [ 5, 15 ], "chance": 50, "repeat": [ 1, 3 ] } ],
+      "place_nested": [
+        {
+          "chunks": [ [ "roof_2x2_golf", 20 ], [ "roof_2x2_utilities_b", 50 ], [ "null", 50 ], [ "roof_3x3_wine", 20 ] ],
```
Comment (ZhilkinSerg) [inline data/json/mapgen/dispensary.json]: Nested mapgen `roof_3x3_wine` doesn't exist anywhere else in the files.

Comment (curstwist) [reply data/json/mapgen/dispensary.json]: Sorry, got ahead of myself.  This is dependent on Adds antique store roof #28859 for the roof_3x3_wine mapgen.  I've edited my description.  Let me know if I need to link them more explicitly.

**Comment by curstwist** [comment]
> 
> 
> I can't find a stairs or ladder up to the roof on the `dispensary` location. Is this intentional?

Yes, gutters can also be used for roof access.  I'm trying to vary access pathways. I try to keep certain nested mapgens limited to places with ladders/stairs like the sandbag last holdout nested chunk, or any that can't be reasonably on a roof without proper access.  I'll probably need to review some of my earlier maps like abstorefront & abandoned_warehouse to make sure access is consistent with nested chunks since this idea came to me a bit late.  With the updated tile set, the t_gutter_downspout is less obvious, the character is north of it in the screenshot.

**Comment by Night-Pryanik** [comment]
I just saw a brazier, a table and chairs up here. How did they get carried there? By gutter?

**Comment by curstwist** [comment]
> 
> 
> I just saw a brazier, a table and chairs up here. How did they get carried there? By gutter?

Realized that too, ha.  Was just deciding between removing or adding access. Think I'll add access and be more careful.

**Pull request merged**